### PR TITLE
kubevirt, sig-ci: add charter and entry in sigs.yaml

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

Adds the charter and sigs.yaml entry for the `sig-ci`. This is done to make clear the responsibilities that are different from `sig-buildsystem`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/cc @brianmcarey @fabiand

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Review by brianmcarey — COMMENTED** [review]
/lgtm

Just one nit - I wasn't able to access the meeting notes when I tried with incognito mode so I am not sure if they are publicly available?

**Comment by dhiller** [comment]
> /lgtm
> 
> Just one nit - I wasn't able to access the meeting notes when I tried with incognito mode so I am not sure if they are publicly available?

No, they aren't. I think we can make that work, while at the same time I am not sure about the details.

**Comment by aburdenthehand** [comment]
@dhiller @brianmcarey We can share it with the kubevirt-dev google group, but I don't think that will allow it to be viewable from anyone. 
We can copy and paste the contents into a new doc owned by the kubevirt account (like the community and other sig docs) to make it more widely viewable, and then update the link here.

**Comment by dhiller** [comment]
> @dhiller @brianmcarey We can share it with the kubevirt-dev google group, but I don't think that will allow it to be viewable from anyone. We can copy and paste the contents into a new doc owned by the kubevirt account (like the community and other sig docs) to make it more widely viewable, and then update the link here.

@aburdenthehand maybe it just makes sense to start with a clean slate here, since there might be RH internal notes present, which we might need to remove first before copying...

**Comment by aburdenthehand** [comment]
@dhiller makes sense. 
I've created this doc for you and copy+pasted your headers: https://docs.google.com/document/d/1pWg6JvjStffoZ0NnVk4ALQOs3MYOdHtBpkLBlJlHTqA/edit#heading=h.mkspjpda1stc
 We can add some Overview, etiquette, etc info afterwards (the [SIG-storage meeting doc](https://docs.google.com/document/d/1mqJMjzT1biCpImEvi76DCMZxv-DwxGYLiPRLcR6CWpE/edit#heading=h.fqe36hzerfrn) is a good example).
it also occurred to me that we will probably want to update the google-meet links that we have listed in this charter. I was not able to join without requesting permission from a non-RH account. We can create a recurring zoom meeting, which also has the added benefit of handling the recordings to be posted to our youtube channel. WDYT?

**Comment by fabiand** [comment]
@dhiller how can we converge aka merge this?

**Comment by dhiller** [comment]
> @dhiller makes sense. I've created this doc for you and copy+pasted your headers: [docs.google.com/document/d/1pWg6JvjStffoZ0NnVk4ALQOs3MYOdHtBpkLBlJlHTqA/edit#heading=h.mkspjpda1stc](https://docs.google.com/document/d/1pWg6JvjStffoZ0NnVk4ALQOs3MYOdHtBpkLBlJlHTqA/edit#heading=h.mkspjpda1stc) We can add some Overview, etiquette, etc info afterwards (the [SIG-storage meeting doc](https://docs.google.com/document/d/1mqJMjzT1biCpImEvi76DCMZxv-DwxGYLiPRLcR6CWpE/edit#heading=h.fqe36hzerfrn) is a good example). it also occurred to me that we will probably want to update the google-meet links that we have listed in this charter. I was not able to join without requesting permission from a non-RH account. We can create a recurring zoom meeting, which also has the added benefit of handling the recordings to be posted to our youtube channel. WDYT?

Sounds good to me.

File: sigs.yaml [hunk 1]
```diff
@@ -104,6 +104,39 @@ sigs:
...
+          name: "Daniel Hiller"
+          company: Red Hat
+      meetings:
+      - description: KubeVirt CI Taskforce Sync meeting
+        day: Monday
+        time: "09:00"
+        tz: UTC
+        frequency: weekly
+        url: https://meet.google.com/pcy-dnin-ojj
+        archive_url: https://docs.google.com/document/d/17eKwt7zaPsEcFrP6hVEz2Bvj_DVg3zJLIUMO0XM7kp4/edit?usp=sharing
```
Comment (aburdenthehand) [inline sigs.yaml:126]:
```suggestion
        archive_url: https://docs.google.com/document/d/1pWg6JvjStffoZ0NnVk4ALQOs3MYOdHtBpkLBlJlHTqA/edit
```

File: sigs.yaml [hunk 2]
```diff
@@ -104,6 +104,39 @@ sigs:
...
+        - github: dhiller
+          name: "Daniel Hiller"
+          company: Red Hat
+      meetings:
+      - description: KubeVirt CI Taskforce Sync meeting
+        day: Monday
+        time: "09:00"
+        tz: UTC
+        frequency: weekly
+        url: https://meet.google.com/pcy-dnin-ojj
```
Comment (aburdenthehand) [inline sigs.yaml:125]:
```suggestion
        url: https://zoom.us/j/96583958896
```
I've set this to an hour. Please let me know if it should be shorter (or longer..?)

File: sigs.yaml [hunk 3]
```diff
@@ -104,6 +104,39 @@ sigs:
...
+        archive_url: https://docs.google.com/document/d/17eKwt7zaPsEcFrP6hVEz2Bvj_DVg3zJLIUMO0XM7kp4/edit?usp=sharing
+      - description: KubeVirt tests quarantine catch-up
+        day: Wednesday
+        time: "08:45"
+        tz: UTC
+        frequency: weekly
+        url: https://meet.google.com/obu-gcyx-afz
+        archive_url: https://docs.google.com/document/d/17eKwt7zaPsEcFrP6hVEz2Bvj_DVg3zJLIUMO0XM7kp4/edit?usp=sharing
```
Comment (aburdenthehand) [inline sigs.yaml:133]:
```suggestion
        url: https://zoom.us/j/94552009485
        archive_url: https://docs.google.com/document/d/1pWg6JvjStffoZ0NnVk4ALQOs3MYOdHtBpkLBlJlHTqA/edit
```
Also set to an hour

**Review by aburdenthehand — COMMENTED** [review]
Adding the new agenda and zoom links

File: sigs.yaml [hunk 4]
```diff
@@ -129,6 +129,39 @@ sigs:
...
+      leadership:
+        chairs:
+        - github: brianmcarey
+          name: "Brian Carey"
+          company: Red Hat
+        - github: dhiller
+          name: "Daniel Hiller"
+          company: Red Hat
+      meetings:
+      - description: KubeVirt CI Taskforce Sync meeting
```
Comment (dhiller) [inline sigs.yaml:145]: @aburdenthehand qq wrt to the Google Calendar: can I create the Google Calendar agenda item myself? Or do you need to do this?

Comment (aburdenthehand) [reply sigs.yaml:145]: IIUC it needs to be done using the KubeVirt account for it to show up on the shared calendar and mirrored on the CNCF calendar.

**Comment by aburdenthehand** [comment]
/approve
I don't think we're waiting on anything except maybe @brianmcarey for a quick re-review?

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Pull request merged**